flexy-druid/src/test/resources/spring/applicationContext-test.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="
+		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+
+  <context:component-scan base-package="com.vladmihalcea.flexypool.config"/>
+  <!-- Xml configuration -->
+  <!--<import resource="classpath:spring/applicationContext-xml-config.xml"/>-->
+
+  <import resource="classpath:spring/applicationContext-local-resource.xml"/>
+
+  <bean id="poolingDataSource" class="com.alibaba.druid.pool.DruidDataSource" destroy-method="close">
+    <!-- 基本属性 url、user、password -->
```

Comment:
Can you try to translate the Chinese description to English? Thanks.

Reply:
Sorry, I forgot it.